cas/worker/tests/running_actions_manager_test.rs

```diff
@@ -1106,14 +1106,19 @@ exit 0
                 .await?
                 .write_all(TEST_WRAPPER_SCRIPT_CONTENT.as_bytes())
                 .await?;
+            #[cfg(target_family = "unix")]
+            test_wrapper_script_handle
+                .as_writer()
+                .await?
+                .as_mut()
+                .set_permissions(Permissions::from_mode(0o755))
+                .await?;
```

Comment:
Is there a reason we do not need to include error handling here for the futures like we did in other places with:

```rs 
.await?
.err_tip(|| "Could not sync file");
```